src/mlpack/core/data/detect_file_type.hpp

```diff
@@ -23,8 +25,7 @@ namespace data {
  *
  * @param type Type to get the logical name of.
  */
-std::string GetStringType(const arma::file_type& type);
-
+std::string GetStringType(const file_type& type);
```

Comment:
```suggestion
std::string GetStringType(const file_type& type);

```
I think the code is a little easier to read if we keep the space here. :+1: